### ISSUE TITLE
Lina arcana portrait bug

### ISSUE THREAD

**Issue by mwacc1234** [ValveSoftware/Dota2-Gameplay#10923]
### Cosmetic Item Name

Fiery Soul of the Slayer

### Description

Most of the time, when Lina gets hexed by someone (such as Lion's hex ability, Shaman's hex ability, Scythe of Vyse), her portrait bugs. See the image attached for reference. Thank you!

### Example Match ID

N/A

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/118600874/18223e72-9556-40a4-8bc5-171a800a6104)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/118600874/99c60eb4-b1fa-4b3a-9915-bbb69eb92cd3)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/118600874/80d1010c-7d36-44cf-b603-c657a267773b)

**Comment by Astiberon** [comment]
> I don't think this is a bug. Because you are in "hex". It's the same when you're dead and your killer's portrait is a frog, but that's just because the hero who killed you got the "hex" effect.

The author means that after the Hex effect ends on Lina, her portrait breaks.
![вапв](https://github.com/ValveSoftware/Dota2-Gameplay/assets/107125193/187ce729-7f8a-4dd1-bab2-f0f4a0ac6c46)

**Comment by motherofautists** [comment]
> > I don't think this is a bug. Because you are in "hex". It's the same when you're dead and your killer's portrait is a frog, but that's just because the hero who killed you got the "hex" effect.
> 
> The author means that after the Hex effect ends on Lina, her portrait breaks. ![вапв](https://user-images.githubusercontent.com/107125193/263045704-187ce729-7f8a-4dd1-bab2-f0f4a0ac6c46.png)

Oh well. 
Words matter.

I know that the Silencer has a funny portrait bug. When his head seems very small, as if he is very far away. 🗿

**Comment by Astiberon** [comment]
> I know that the Silencer has a funny portrait bug. When his head seems very small, as if he is very far away. 🗿
Yes you are right. I posted this bug, you can find it under number 799.
